pkg/network/types/connection_key.go

```diff
@@ -5,7 +5,10 @@
 
 package types
 
-import "github.com/DataDog/datadog-agent/pkg/process/util"
```

Comment:
Could you move this to `pkg/network/protocols` please? Its only being used by USM at the moment.